package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@firebolt-js/sdk",
-  "version": "0.9.1-next.1",
+  "version": "0.10.0",
```

Comment:
I don't think you need this, since your commit has "feat:" in it, then when you merge to next it should auto do this.